File: FE_ON/src/app/shared/service/rating-image-service/rating-image.service.ts
import { Injectable } from '@angular/core';
import {RatingImageApiService} from "./rating-image-api.service";

@Injectable({
  providedIn: 'root'
})
export class RatingImageService {


  constructor( private ratingImgService: RatingImageApiService ) { }

  creatRatingImg( rateImg: any ){
     return this.ratingImgService.createRatingImage(rateImg) ;
  }

  getRatingImgByIdRating( list:any ){
     var a = ''
     for( let x of list ){
        a += 'rateImg=' + x.id + '&'
     }

     return this.ratingImgService.getRatingImgByIdRating(a) ;
  }
}

File: BE/src/main/java/nem/com/service/RatingImageService.java
package nem.com.service;

import nem.com.entity.RatingImages;

import java.util.List;

public interface RatingImageService {

    RatingImages createRatingImage( RatingImages ratingImages );

    List<RatingImages> getAll() ;

    List<RatingImages> getRatingImgByIdRating( Long[] id );
}
